### PR TITLE
chore: allow users with admin role to download audit log from UI

### LINKED ISSUE #586: Add audit log to UI (stream RPC)

### PR DESCRIPTION
**Pull request by DmitriyMV**
Currently, there is no way to write to file using stream-like API, so we have to concatenate the full log in memory, before we save it to the file system. In practice, if this proves to be problematic we can always redo the logic using standard http file download.

Closes #586

### CODE REVIEW

File: frontend/src/views/omni/Overview/Overview.vue [hunk 1]
```diff
@@ -170,6 +173,12 @@ const hasRoleNone = computed(() => {
   return !role || role === RoleNone;
 });
 
+const hasRoleAdmin = computed(() => {
+  const role = currentUser.value?.spec?.role
```
Comment (Unix4ever) [inline frontend/src/views/omni/Overview/Overview.vue:177]: I'd suggest to create a separate flag like `canDownloadAuditLog` in the currentUser resource.

File: hack/release.toml [hunk 2]
```diff
@@ -7,3 +7,19 @@ match_deps = "^github.com/((talos-systems|siderolabs)/[a-zA-Z0-9-]+)$"
 
 pre_release = true
 previous = "v0.40.0"
+
+[notes]
+
+  [notes.extensions]
```
Comment (Unix4ever) [inline hack/release.toml:13]: :heart:

Comment (DmitriyMV) [reply hack/release.toml]: Yeah, sorry, updated to proper name.

File: frontend/src/views/omni/Overview/Overview.vue (hunk 1 above)
Comment (DmitriyMV) [reply frontend/src/views/omni/Overview/Overview.vue]: Done!

**Review by Unix4ever — APPROVED** [review]

**Pull request merged**